vehicle/mkvii/software/dashboard/dashboard.c

```diff
@@ -0,0 +1,207 @@
+/*
+    Header:
+    Contains code for the dashboard board
+    Function: Responsible for BMS and IMD LED Indicators - DONE
+                -listening for CAN messages and controlling LED appropriately
+            : Start button + LED - DONE
+                - based on input off of the physical start button before
+   entering RTD : Interface with LED Bars Boards - TODO
+                - listening for CAN messages and outputting over I2C
+            : Implement other LEDs (brake, LV, HV) - DONE
+                - CAN
+            : Implement power cycle logic - DONE
+            : Implement RTD Buzzer - DONE
+*/
+
+#include "dashboard_config.h"
+#include "libs/adc/api.h"
+#include "libs/gpio/api.h"
+#include "libs/timer/api.h"
+#include "vehicle/mkv/software/dashboard/can_api.h"
+#include <util/delay.h>
+
+
+#include "projects/btldr/btldr_lib.h"
+#include "projects/btldr/git_sha.h"
+#include "projects/btldr/libs/image/api.h"
+#include <stdio.h>
+
+/*
+ * Required for btldr
+ */
+image_hdr_t image_hdr __attribute__((section(".image_hdr"))) = {
+    .image_magic = IMAGE_MAGIC,
+    .git_sha = STABLE_GIT_COMMIT,
+};
+
+#define BMS_FAULT_NONE (0)
+#define AIR_STATE_TS_ACTIVE (4)
+
+volatile bool START_BUTTON_STATE;
+bool HV_STATE;
+bool BRAKE_PRESSED;
+bool THROTTLE_PRESSED;
+
+volatile bool send_can;
+volatile int buzzer_counter = 0;
+int dashboard_counter = 0;
+
+// Start Button interrupt & final ReadyToDrive check
+void pcint1_callback(void) {
+    START_BUTTON_STATE = !gpio_get_pin(START_BTN);
```

Comment:
We don't need a separate tracking variable here; we can just use the `dashboard.start_button_state` struct field for this.